ocp-public.repo

```diff
@@ -0,0 +1,10 @@
+[ocp-412-el8-beta-public]
```

Comment:
Hello @cgwalters,

looking better at the PR, I see you're including the public mirror for the rpms (cri-o, clients, hyperkube...). That repo is only available for the x86_64 arch, though. We relied on the builds from the artifacts and hyperkube images (both from Prow and the payload tekton pipeline) to get those packages till now. Any plan to have the same repo for arm64 exposed?

Reply:
The problem seems to be here
https://github.com/openshift/aos-cd-jobs/blob/70b48b971e244c931a93185f10910d5acd11ac20/jobs/build/publish-rpms/Jenkinsfile#L47

But it's not obvious to me what's in that bucket etc.

Reply:
Are those jobs running on x86-only agents? If yes, we'd need other changes [here](https://github.com/openshift/aos-cd-jobs/blob/70b48b971e244c931a93185f10910d5acd11ac20/jobs/build/publish-rpms/collect_deps.py), right?

+1 for @travier and @LorbusChris from my side